odd-platform-ui/src/components/Overview/Overview.tsx

```diff
@@ -20,6 +21,10 @@ const Overview: React.FC = () => {
     page: 1,
     size: 30,
   });
+  const { data: appInfo } = useAppInfo();
+  const isShowOwnerAssociation = Boolean(
+    appInfo?.authType && appInfo.authType !== 'DISABLED'
+  );
```

Comment:
**nit**: it's a good practice to use useMemo hook for that types of variables but it's not necessary in that particular case. And it seems we don't need data type coercion here
```suggestion
  const isShowOwnerAssociation = useMemo(() =>
    appInfo?.authType && appInfo.authType !== 'DISABLED', 
  [appInfo?.authType])
```